docs: simplify OIDC config instructions (#16266)
coder/coder: docs/admin/users/oidc-auth.md

## Step 2: Configure Coder with the OpenID Connect credentials

> Set the following environment variables on your Coder deployment and restart Coder:

Reviewer: optional

```suggestion
Set the following environment variables in your Coder deployment's `/etc/coder.d/coder.env` file, then restart Coder:
```

Author: What I am trying to do is make all the instructions common. The `env` file doesn't apply for K8 s and docker based installations.
